WordPress/Classes/ViewRelated/Blog/SharingAuthorizationHelper.m

```diff
@@ -231,32 +231,25 @@ - (void)fetchKeyringConnectionsForService:(PublicizeService *)pubServ
 
  @param keyringConnections: An array of `KeyringConnection` instances.
  */
-- (void)showAccountSelectorForKeyrings:(NSArray *)keyringConnections withKeyringAccountHelper:(KeyringAccountHelper *)keyringAccountHelper
+- (void)showAccountSelectorForKeyrings:(NSArray *)keyringConnections
 {
     NSParameterAssert([[keyringConnections firstObject] isKindOfClass:[KeyringConnection class]]);
 
     SharingAccountViewController *controller = [[SharingAccountViewController alloc] initWithService:self.publicizeService
                                                                            connections:keyringConnections
-                                                                   existingConnections:[self connectionsForService]
-                                                                                keyringAccountHelper: keyringAccountHelper];
+                                                                   existingConnections:[self connectionsForService]];
     controller.delegate = self;
 
     // Set the view controller stack vs push so there is no back button to contend with.
     // There should be no reason for the user to click back to the authorization vc.
     [self.navController setViewControllers:@[controller] animated:YES];
 }
 
--(void)showAlertFromConnectionsWithKeyringAccountHelper:(KeyringAccountHelper *)keyringAccountHelper
+-(void)showAlertFromConnectionsWithValidationError:(ValidationError *)validationError
```

Comment:
Since this is only called once, do you think it's a good idea to just get rid of this and go straight with calling

```objc
    [weakSelf.delegate sharingAuthorizationHelper:self
                 requestToShowValidationError:validationError
                           fromViewController:self.navController];
```

... in `fetchKeyringConnectionsForService` above?